alpine3.14/Dockerfile

```diff
@@ -32,7 +32,9 @@ RUN apk add --no-cache --virtual .build-deps \
                 bash \
                 jq \
                 flex \
-                bison
+                bison \
+                autoconf \
```

Comment:
`autoconf` is already present [above](https://github.com/emqx/emqx-builder/pull/29/files#diff-62124d8ddd7e4cc0d609699753c6167fe738beb194715dd61e3127e8aa9f9b64R15).

Reply:
Fixed in added commit